### ISSUE TITLE
Duplicated icons show up

### ISSUE THREAD

**Issue by varac** [Alexays/Waybar#306]
Sometimes icons duplicates show up in my bar. 
It happens when I switch from my dual-display setup at home to my dual display setup at work.
The bar on the external display is fine but the bar on the laptop shows duplicate icons. All icons respond to clicks.
The fun fact is that I cannot take a screenshot of it with `grim -g "$(slurp)" /tmp/screenshot.png`, the resulting screenshot is either blank or shows some unrelated section of the screen.
So went ahead and took a screenshot with my mobile (duh!).

![IMG_20190508_105854 cleaned](https://user-images.githubusercontent.com/488213/57365530-848e0600-7185-11e9-993e-25c577ae84ea.jpg)

**Comment by varac** [comment]
This is the relevant section from the logs, at 10:28 I connected to my office display:

```
Mai 08 09:34:47 box systemd[3305]: waybar.service: Succeeded.
Mai 08 09:34:48 box waybar[29834]: Resources files: /home/varac/.config/waybar/config, /home/varac/.config/waybar/style.css
Mai 08 09:34:48 box waybar[29834]: Bar configured (width: 1366, height: 25) for output: LVDS-1
Mai 08 09:34:48 box waybar[29834]: Bar configured (width: 1680, height: 25) for output: HDMI-A-3
Mai 08 09:34:48 box waybar[29834]: An object is already exported for the interface org.kde.StatusNotifierWatcher at /StatusNotifierWatcher
Mai 08 09:50:34 box waybar[29834]: Bar removed from output: HDMI-A-3
Mai 08 10:28:21 box waybar[29834]: Bar configured (width: 1920, height: 25) for output: HDMI-A-1
Mai 08 10:57:51 box waybar[29834]: Requested width: 1366 exceeds the minimum width: 1384 required by the modules
Mai 08 10:57:51 box waybar[29834]: Bar configured (width: 1384, height: 25) for output: LVDS-1
```

btw, a restart of waybar fixes the issue.

**Comment by Alexays** [comment]
Fixed with b54160e02f4391427004d44e0452e57fc0529afa

**Comment by varac** [comment]
thx @Alexays !
